packages/components/src/pagination-control/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { __, sprintf } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { times } from 'lodash';
+import './style.scss';
+
+interface Props {
+	currentPage: number;
+	numberOfPages: number;
+	setCurrentPage: ( page: number ) => void;
+	classNames?: string | string[];
+}
+
+const PaginationControl: React.FunctionComponent< Props > = ( props ) => {
+	const { currentPage, numberOfPages, setCurrentPage, classNames, children } = props;
+
+	const classes = classnames( 'pagination-controls', classNames?.toString()?.split( ',' ) );
+
+	return (
+		<ul className={ classes } aria-label={ __( 'Guide controls' ) }>
+			{ times( numberOfPages, ( page ) => (
```

Comment:
I'd call `page` index here, since that's what it is.

Reply:
👍